src/Files.App/TemplateSelectors/BaseTemplateSelector.cs

```diff
@@ -6,23 +6,15 @@ namespace Files.App.TemplateSelectors
 	internal abstract class BaseTemplateSelector<TItem> : DataTemplateSelector
 	{
 		protected sealed override DataTemplate SelectTemplateCore(object item)
-		{
-			return SelectTemplateCore((TItem?)item);
-		}
+			=> SelectTemplateCore((TItem?)item);
 
 		protected sealed override DataTemplate SelectTemplateCore(object item, DependencyObject container)
-		{
-			return SelectTemplateCore((TItem?)item, container);
-		}
+			=> SelectTemplateCore((TItem?)item, container);
 
 		protected virtual DataTemplate SelectTemplateCore(TItem? item)
-		{
-			return base.SelectTemplateCore(item);
-		}
+			=> base.SelectTemplateCore(item);
 
 		protected virtual DataTemplate SelectTemplateCore(TItem? item, DependencyObject container)
-		{
-			return base.SelectTemplateCore(item, container);
-		}
+			=> base.SelectTemplateCore(item, container);
```

Comment:
I'm not sure you should use arrow functions there.
@d2dyno1 what do you think?

Reply:
Yep, these shouldn't be one-liners